resources/js/components/Customizer/PrizePacks.vue

```diff
@@ -123,23 +138,43 @@
 </template>
 
 <script>
+import Toggle from "../Toggle.vue";
 import Select from "../Select.vue";
 
 export default {
   components: {
-    Select
+    Toggle: Toggle,
+    Select: Select
+  },
+  data() {
+    return {
+      customPrizePacks: false
+    }
+  },
+  created() {
+    this.customPrizePacks = this.$store.state.context.settings["customPrizePacks"]
```

Comment:
I believe we generally avoid local state and use computed's introspecting on the store